### PR TITLE
[ci] Fix remote project handling in update repositories

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

**Comment by bgeuken** [comment]
@adrianschroeter Please review.

**Comment by adrianschroeter** [comment]
+1

**Comment by hennevogel** [comment]
How is this [ci]?

**Comment by bgeuken** [comment]
Oh, right. Let me change it

File: src/api/app/models/project.rb [hunk 1]
```diff
@@ -677,15 +677,16 @@ def update_one_repository_without_path(repo)
...
-      end
-      # FIXME: Repository.find_by_project_and_name uses a not_remote scope which makes this check obsolete
-      unless target_repo.remote_project_name.nil?
-        raise SaveError.new("Can not use remote repository as release target '#{rt['project']}/#{rt['repository']}'")
+      if Project.find_by(name: rt['project']).is_remote?
+        raise SaveError, "Can not use remote repository as release target '#{rt['project']}/#{rt['repository']}'"
+      else
+        target_repo = Repository.find_by_project_and_name(rt['project'], rt['repository'])
+        if target_repo
+          current_repo.release_targets.new(:target_repository => target_repo, :trigger => rt['trigger'])
```
Comment (mdeniz) [inline src/api/app/models/project.rb]: Use new syntax for the params hash

**Pull request merged**